v2/pkg/engine/resolve/resolvable.go

```diff
@@ -64,6 +64,8 @@ type ResolvableOptions struct {
 	ApolloCompatibilitySuppressFetchErrors          bool
 	ApolloCompatibilityReplaceUndefinedOpFieldError bool
 	ApolloCompatibilityReplaceInvalidVarError       bool
+
+	ApolloRouterCompatibilityReplaceInvalidVarError bool
```

Comment:
Looks like it is not needed here

Reply:
It's not on the hot path for the router, but it is used in this one other call-site for `NewVariablesValidator`.

Reply:
you never call variables validator from the resolvable

Reply:
Removed, `NewVariablesValidator` is called in `execution_engine.go` via `(e *ExecutionEngine)`'s `ResolvableOptions`, but this is not used ever within the repository and we shouldn't need to introduce Apollo Router compatibility for anyone consuming exported functions from `graphql-go-tools` unless they want it.